tests/gnrc_dhcpv6_client/Makefile

```diff
@@ -17,9 +17,9 @@ ifneq (,$(filter native,$(BOARD)))
   # Has to be provided here and not in Makefile.dep, so TERMFLAGS are properly
   # configured
   USEMODULE += netdev_default
-  IFACE ?= tapbr0
+  export IFACE ?= tapbr0
```

Comment:
Mhh then we need to get it to the config checker script in #16797 too somehow as well. Which, given that this is called in the bowels of the make system, seems way more complicated.

Reply:
That's also a target no? anyway it's a nitpick because this export does not have a performance impact. I was just brainwashed into trying to avoid exports.

Reply:
Right. I forgot, that you can do that with make. Will remove the export!

Reply:
Done.